sdk/monitor/azure-monitor-query-metrics/README.md

```diff
@@ -0,0 +1,291 @@
+    System.out.println("Metrics: " + queryResult.getMetrics().size() + "\n");
+    // print out the metrics for each query result
+    for (MetricResult metric : queryResult.getMetrics()) {
+        System.out.println("Metric Name: " + metric.getMetricName());
+        System.out.println("Unit: " + metric.getUnit());
+        System.out.println("Time Series Count: " + metric.getTimeSeries().size());
+
+        metric.getTimeSeries().forEach(timeSeries -> {
+            timeSeries.getValues().forEach(value -> {
+                System.out.println("Timestamp: " + value.getTimeStamp()
+                    + ", Average: " + value.getAverage()
+                    + ", Count: " + value.getCount());
+            });
+        });
+    }
+}
+```
+
+## Troubleshooting
+
+See our [troubleshooting guide](https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/monitor/azure-monitor-query/TROUBLESHOOTING.md)
+for details on how to diagnose various failure scenarios.
+
+## Next steps
+
+To learn more about Azure Monitor, see the [Azure Monitor service documentation][azure_monitor_overview].
+
+## Contributing
+
+This project welcomes contributions and suggestions. Most contributions require you to agree to a [Contributor License Agreement (CLA)][cla] declaring that you have the right to, and actually do, grant us the rights to use your contribution.
+
+When you submit a pull request, a CLA-bot will automatically determine whether you need to provide a CLA and decorate
+the PR appropriately (e.g., label, comment). Simply follow the instructions provided by the bot. You will only need to
+do this once across all repos using our CLA.
+
+This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For more information, see
+the [Code of Conduct FAQ][coc_faq] or contact [opencode@microsoft.com][coc_contact] with any additional questions or
+comments.
+
+
+<!-- LINKS -->
+[azure_identity]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/README.md
+[azure_monitor_create_using_portal]: https://learn.microsoft.com/azure/azure-monitor/logs/quick-create-workspace
+[azure_monitor_overview]: https://learn.microsoft.com/azure/azure-monitor/overview
+[azure_subscription]: https://azure.microsoft.com/free/java
+[changelog]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/monitor/azure-monitor-query/CHANGELOG.md
+[custom_subdomain]: https://learn.microsoft.com/azure/cognitive-services/authentication?tabs=powershell#create-a-resource-with-a-custom-subdomain
+[DefaultAzureCredential]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/identity/azure-identity/README.md#defaultazurecredential
+[jdk_link]: https://learn.microsoft.com/java/azure/jdk/?view=azure-java-stable
+[kusto_query_language]: https://learn.microsoft.com/azure/data-explorer/kusto/query/
+[log_levels]: https://github.com/Azure/azure-sdk-for-java/blob/main/sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java
```

Comment:
Fix all the links to use the new library reference. Go through the whole readme and fix all links and references to old names/links/library.